spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -269,38 +277,38 @@
   describe "#build_creator_contributor_json" do
     context 'personal record without user in digital hub, with proper name formatting' do
       it 'assigns' do
-        expect(converter.send(:build_creator_contributor_json, non_user_properly_formatted).with_indifferent_access).to eq(personal_creator_without_user_json)
+        expect(invenio_rdm_record_converter.send(:build_creator_contributor_json, non_user_properly_formatted).with_indifferent_access).to eq(personal_creator_without_user_json)
```

Comment:
Removing the blank file check was breaking this since there was no GenericFile object being passed in. I'm starting to wonder if at some point blank files were no longer an issue, but it remained in the logic to keep specs green.